definitions/infra-awskinesisanalyticsapplication/golden_metrics.yml

```diff
@@ -0,0 +1,45 @@
+millisBehindLatest:
+  title: Processing lag
+  query:
+    select: average(provider.millisBehindLatest.Average) * 1000
+    from: AwsKinesisAnalyticsApplicationSample
+    where: provider='AwsKinesisAnalyticsApplication'
+    facet: entityName
+    eventId: entityGuid
+  unit: SECONDS
```

Comment:
unless I'm missing something we don't have units in golden metrics

Reply:
updated, thanks